data/50-default.json

```
@@ -91,7 +91,7 @@
     "serial": 2019102100,
     "ref-kind": "app",
     "collection-id": "com.endlessm.Apps",
-    "remote": "eos-apps",
+    "remote": "flathub",
     "name": "com.hack_computer.Clubhouse",
     "branch": "eos3",
```

Are we using the eos3 branch on flathub as well? Or does this refer to the old branch? I'm also not sure if we should just update the existing entries here as you did or add new ones, if updating existing ones should we also update the serial?

---

Indeed this needs to be changed to:

```suggestion
    "branch": "stable",
```

We actually *don't* want to update the serial. There are 2 cases:

1. User is upgrading from a version which did not already have Clubhouse installed. In that case, we want them to fetch Clubhouse from Flathub, not eos-apps.
2. User is upgrading from a version after this migration has already been applied.
  - If they already have Clubhouse installed, https://github.com/endlessm/eos-boot-helper/pull/324 will rewrite the installed Flatpak to get future updates from Flathub
  - If they don't, then we do not want Clubhouse to be reinstalled